samples/GuestConfiguration/installed-application-linux/azurepolicyset.json

```diff
@@ -0,0 +1,30 @@
+{
+    "properties": {
+        "displayName": "Audit if an application is not installed inside Linux VMs",
+        "description": "This initiative will both deploy the policy requirements and audit if the specified application is not installed inside Linux virtual machines. For more information on Guest Configuration policies, please visit https://aka.ms/gcpol",
+        "parameters": {
+            "applicationName": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Application name"
+                }
+            }
+        },
+        "policyDefinitions": [
+            {
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/fee5cb2b-9d9b-410e-afe3-2902d90d0004"
+            },
+            {
+                "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/4d1c04de-2172-403f-901b-90608c35c721",
+                "parameters": {
+                    "applicationName": {
+                        "value": "Application name"
+                    }
+                }
+            }
+        ]
+    },
+    "id": "/providers/Microsoft.Authorization/policySetDefinitions/c937dcb4-4398-4b39-8d63-4a6be432252e",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "name": "guestconfig-installed-application-linux"
```

Comment:
Samples should not have id, type, or name

Reply:
Fixed.